### ISSUE TITLE
Cannot reproduce BRNet detection results

### ISSUE THREAD

**Issue by xiyuchenji** [alperkesen/3D-visual-grounding#1]
Dear Alper,thank you for your flexible code.I meet a large performance gap with the paper of BRNet(CVPR 2021) when I only use brnet.

For example on the scannet dataset,with Adam optimizer,learning_rate=0.005,lr_decay_rates='0.1,0.1,0.1', lr_decay_steps='120,160,200', max_epoch=220,

I got AP_0.25,    AR_0.25 
        0.582306    0.795881
On the scannet dataset,with AdamW optimizer, learning_rate=0.008,  lr_decay_rates='0.1,0.1', lr_decay_steps='120,160', max_epoch=220

I got  AP_0.25,    AR_0.25 
         0.593602   0.794823

There is a large performance gap between the above result and the paper of BRNet(0.661 for mAP@0.25) , which reall puzzle me. Do you have some suggestions? I am looking forward for your reply,thank you!

**Comment by alperkesen** [comment]
Hi @xiyuchenji,

Our configurations may not be completely the same as [original BRNet implementation](https://github.com/cheng052/BRNet). So, it makes sense that there may be a gap between this implementation and original one. 

For example, (1) in BRNet implementation/paper, [cosine annealing strategy](https://github.com/cheng052/BRNet/blob/master/configs/_base_/schedules/schedule_cos.py) was used for learning rate decay, (2) AdamW optimizer was used in BRNet with learning rate of 0.005 (3) we didn't change the FAR_THRESHOLD and left it the same as in the original ScanRefer implementation (Check [lib/loss_helper.py](https://github.com/alperkesen/3D-visual-grounding/blob/master/lib/loss_helper.py)) although it was different in BRNet implementation (Check [BRNet config](https://github.com/cheng052/BRNet/blob/master/configs/_base_/models/brnet.py)).

My suggestion would be training the original BRNet implementation with default settings and make sure that you can reproduce the results. Then, you can check the differences between this code and original BRNet and add/change these configurations on this code and recheck the results.
